hocuspocus-server/src/setMark.ts

```diff
@@ -23,7 +24,75 @@ export function setMark(
 ) {
   // needed to get the pmSchema
   // if you use a BlockNote custom schema, make sure to pass it to the create options
-  const editor = ServerBlockNoteEditor.create();
+  const editor = ServerBlockNoteEditor.create({
```

Comment:
let's try with just passing `comments` to the editor options before merging this, I think that should work right?

Reply:
I had tried this, but it wanted a ydoc I think. Maybe your [PR](https://github.com/TypeCellOS/BlockNote/pull/1520) will help this